.claude/commands/jira-task.md

```
@@ -0,0 +1,173 @@
+
+```bash
+# .env 또는 환경 변수
+JIRA_URL=https://your-company.atlassian.net
+JIRA_EMAIL=your-email@company.com
+JIRA_API_TOKEN=your-api-token
+JIRA_PROJECT_KEY=MEMBER
+```
+
+### API Token 발급
+
+1. https://id.atlassian.com/manage/api-tokens
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)